site/content/docs/include/sidecar-config.en.md

```diff
@@ -2,8 +2,7 @@
 <a id="port" href="#port" class="field">`port`</a> <span class="type">Integer</span>  
 Port of the container to expose (optional).
 
-<a id="image" href="#image" class="field">`image`</a> <span class="type">String</span>  
-Image URL for the sidecar container (required).
+{% include 'image-config.en.md' %}
```

Comment:
Do we not need to field for `image`? Like it could be a string or a map I assume

Reply:
I missed it 🤦. It should be there.